mlflow/projects/__init__.py

```diff
@@ -534,8 +534,13 @@ def _get_docker_command(image, active_run, docker_args=None, volumes=None, user_
     cmd = [docker_path, "run", "--rm"]
 
     if docker_args:
-        for key, value in docker_args.items():
-            cmd += ['--' + key, value]
+        for name, value in docker_args.items():
+            # Passed just the name as boolean flag
+            if isinstance(value, bool) and value:
+                cmd += ['-' + name]
```

Comment:
I think we need to use a double hyphen for long options (`name` >= 2 chars) and a single hyphen for short options (`name` = 1 char)